exporter/signalfxexporter/config.go

```diff
@@ -24,17 +23,24 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk"
 )
 
-const (
-	translationRulesConfigKey = "translation_rules"
-)
+type translationRulesConfig struct {
```

Comment:
This mimics the original `translation_rules` config option, but is not reachable by user configuration. It's only used by `defaultTranslationRules` to parse the YAML into a `[]translation.Rule` object.